app/scripts/controllers/metamask-notifications/metamask-notifications.ts

```diff
@@ -428,10 +461,11 @@ export class MetamaskNotificationsController extends BaseController<
   public async checkTriggersPresenceByGroup(): Promise<
```

Comment:
I've kept this method, but lets not use this yet. I'm not confident in our implementation of group trigger enable/disable.

Doing an introspection and "guessing" if a filter should be enabled/disabled is really hard, especially breaks down when you also need to consider any new notifications we will add in the future.

Proposal: It might be easier to have a new User Storage entry to say if a filter is enabled/disabled instead of introspecting.